### DISCUSSION TITLE
Share files between pod replicas in different nodes

### DISCUSSION THREAD

**Discussion by cgx-system** [longhorn/longhorn#8264] [Q&A]
Hi,
I'm new to Longhorn and I'm loking for a way to share files between deployment replicas that can be on different nodes of my k8s cluster.
I have a deployment YAML file with : 
- a PersistentVolumeClaim to a longhorn StorageClass volume, in ReadWriteMany accessModes
- a Deployment of 3 replicas of a pod with the previously declared PVC and the same volume name

The replicas are deployed on different nodes and the Volume is mounted on them, but they don't share the same files.
When I write a file in the mounted volume on one pod, I can't read it on the other pods, so I guess each pod has its own volume mounted without replication of the files between them.

How can I achieve persistent file sharing between pods in different nodes ? Is it possible with Longhorn ?

Thanks

**Comment by ChanYiLin** [comment]
Hi @cgx-system 
We do have read write many volume: https://longhorn.io/docs/1.6.1/nodes-and-volumes/volumes/rwx-volumes/
Can you help to verifiy that all your deployment replicas are mounted with the same volume?
And can you help to verify that there is a `share-manager` pod running in the `longhorn-system` namespace?
Thanks!
